content/docs/start/index.md

```diff
@@ -49,13 +49,13 @@ Changes to be committed:
 $ git commit -m "Initialize DVC"
 ```
 
-DVC features can be grouped into layers. We'll explore them one by one in the
-next few sections:
+DVC features can be grouped into functional components. We'll explore them one
+by one in the next few sections:
 
-- [**Data versioning**](/doc/start/data-versioning) is the core part of DVC for
-  large files, datasets, machine learning models versioning and efficient
-  sharing. We'll show how to use a regular Git workflow, without storing large
-  files with Git. Think "Git for data".
+- [**Data versioning**](/doc/start/data-versioning) is the base layer of DVC. It
```

Comment:
Again, I don't understand the reason for this change. Not obvious, sounds repetitive.

Reply:
The main change here is "core part" -> "base layer". I was reviewing the usage of terms around "feature". We used "layers" before but the concept of "components" (Composable principle) is more appropriate (this came from reading Hashicorp's Tao).

Reply:
It was more repetitive before: "Data versioning is the core part of DVC for large file versioning..."

But let me re-work this to fix the language with a minimal change... ⌛

Reply:
OK changed it again. I separated the part about "efficient sharing" because it was making the first sentence too complicated. PTAL

> p.s. I think the repetitiveness was inserted recently by a SEO PR now that I think about it... Jeremy wanted to boost the term "ML model versioning". I guess it's too forced to have that in this doc.